### DISCUSSION TITLE
UDP Icecandidates

### DISCUSSION THREAD

**Discussion by danrossi** [OvenMediaLabs/OvenMediaEngine#1654] [Help]
The example server config is forcing TCP ice candidates. Requiring "transport=tcp" param on the connection. If I do <TcpForce>false</TcpForce> in the config a connection cannot be  made.  The iceconnection disconnects with a " Iceconnection Closed" message in the ovenplay demo at the very least. How to get UDP ice candidates working ? The config looks like

```
<IceCandidates>
					<IceCandidate>${env:OME_HOST_IP:*}:${env:OME_WEBRTC_CANDIDATE_PORT:10000-10004/udp}</IceCandidate>
					<TcpRelay>${env:OME_HOST_IP:*}:${env:OME_WEBRTC_TCP_RELAY_PORT:3478}</TcpRelay>
					<!--<TcpForce>true</TcpForce>-->
					<TcpForce>false</TcpForce>
					<TcpRelayWorkerCount>1</TcpRelayWorkerCount>
				</IceCandidates>
```

**Comment by getroot** [comment]
It should work fine even with TcpForce set to false. Check if external players can access your OME_HOST_IP:OME_WEBRTC_CANDIDATE_PORT/udp. If you are using Docker, check if the port is connected and not blocked by a firewall.
